BowlingGame/OpenFrame.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+
+namespace BowlingGame
+{
+
+    // OpenFrame extends the abstract class Frame

```

Comment:
avoid committing notes, we can discuss the reason for this next week

Reply:
Alright cool, I'm guessing it has somethig to do with

Reply:
Code in a business usually has its own design principles, i.e SOLID, well written code that adheres to your place of works principles shouldn't need any comments. We will look at design principles later on